app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -511,42 +513,57 @@ class BrowserTabFragment : Fragment(), FindListener, CoroutineScope {
         }
     }
 
-    private fun externalAppLinkClicked(appLinkCommand: Command.HandleExternalAppLink) {
-        context?.let {
+    private fun openExternalDialog(intent: Intent, context: Context, fallbackUrl: String? = null, showFirstOption: Boolean = true) {
```

Comment:
The purpose of `showFirstOption` wasn't clear to me until I dug into the code and I would like to rename it although it's a tricky one to name. Do you think `useFirstExternalHandler` adds clarity?

Reply:
Agreed! I was struggling trying to find a good name so I'd go with your suggestion. Another option could be 'useFirstActivityFound` or 'useFirstHandlerFound`, any thoughts?

Reply:
They all sound better, happy with all of them.